sonar/modules/users/jsonschemas/users/user-v1.0.0.json

```diff
@@ -2,46 +2,55 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "https://sonar.ch/schemas/users/user-v1.0.0.json",
   "additionalProperties": false,
-  "title": "SONAR user v1.0.0",
+  "title": "User",
   "type": "object",
   "properties": {
     "$schema": {
       "type": "string",
       "default": "https://sonar.ch/schemas/users/user-v1.0.0.json"
     },
     "pid": {
-      "title": "User identifier",
-      "description": "User persistent identifier.",
-      "type": "string"
+      "title": "Identifier",
+      "type": "string",
+      "minLength": 1
     },
     "user_id": {
-      "title": "User account identifier",
-      "description": "User account identifier.",
-      "type": "integer"
+      "title": "Account identifier",
+      "type": "integer",
+      "minLength": 1
     },
     "full_name": {
       "title": "Name",
       "type": "string",
-      "minLength": 2
+      "minLength": 1
     },
     "birth_date": {
       "title": "Date of birth",
+      "description": "Date in yyyy-mm-dd format, ex: 1970-01-01",
       "type": "string",
+      "minLength": 1,
       "format": "date",
-      "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$"
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
     },
     "email": {
       "title": "Email",
       "type": "string",
+      "minLength": 1,
       "format": "email",
-      "pattern": "^.*@.*\\..*$",
-      "minLength": 6
+      "pattern": "^.+@.+\\..+$",
+      "form": {
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {}
+          }
+        }
+      }
     },
     "street": {
       "title": "Street",
       "description": "Street and number of the address, separated by a coma.",
       "type": "string",
-      "minLength": 4
+      "minLength": 1
```

Comment:
Don't think that is mandatory to have a `minLength: 1`. Either the field is empty, either there is necessarily at least one character.